### PR TITLE
Problem: reading genesis time from tendermint make `dev init` fail(fi…

### PR DESCRIPTION
**Pull request by leejw51crypto**
## why , i make this pr
Genesis time is UTC time , and should be later then keypackage time.
also if it's too distant from current utc time, tendermint will not work correctly.

## how i solved 
1. overwrite genesis time with current utc time,
can be disabled by `--no_genesistime_overwrite` option

2. add evidence parameters to dev.json

### CODE REVIEW

**Comment by leejw51crypto** [comment]
checking travis

File: dev-utils/example-dev-conf.json [hunk 1]
```diff
@@ -33,8 +37,8 @@
                 "value": "pPxJH5wUamXcCvSxgywzmANuO0UNR5x3nCbUzsrCeX8="
             },
             {
-                "keypackage": "RklYTUU="
+                "keypackage": "generate keypackage. for e.g.) `ra-sp-server --quote-type Unlinkable --ias-key $IAS_API_KEY --spid $SPID &`,`dev-utils keypackage generate --path ./mls.sgxs --output key.txt` "
```
Comment (tomtau) [inline dev-utils/example-dev-conf.json:40]: there's that `test_dev_config_file` unit test that tries to make sure the "example" file is somewhat valid -- so you may need to include some example keypackage, or at least some base64 payload.

Comment (leejw51crypto) [reply dev-utils/example-dev-conf.json:40]: ok, i'll replace with valid one

**Comment by leejw51crypto** [comment]
ok, i'll add an option `not overwrite time`

**Comment by tomtau** [comment]
> ok, i'll add an option `not overwrite time`

you may need to update the CI script as well, or set this option default to "true"

**Comment by leejw51crypto** [comment]
ok, i'll check drone now

**Comment by leejw51crypto** [comment]
added evidence option, checked with local drone

**Pull request merged**